### PR TITLE
Add final and static

### PR DESCRIPTION
**Pull request by luk98j**

### CODE REVIEW

File: src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+	protected int peekaboo() {
+		if (callCheck())
+			return TOTAL_NUMBER;
+		return numbers[total];
+	}
+
+	public int countOut() {
+		if (callCheck())
+			return TOTAL_NUMBER;
```
Comment (Przydal) [inline src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java]: Variable returned by peekaboo() and callCheck() is not the same number (even if the value is the same).

File: src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 public class IntLinkedList {
 
-	Node last;
-	int i;
+	private Node last;
+	private int i;
```
Comment (Przydal) [inline src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java]: value returned by top() is constant and should be extracted.

File: .gitignore [hunk 3]
```diff
@@ -2,3 +2,5 @@
 .project
 .classpath
 .settings/
+.idea/
```
Comment (Przydal) [inline .gitignore:5]: There was no requirement to amend .gitignore file, please discard changes.

**Review by Przydal — CHANGES_REQUESTED** [review]
Missing few final modificators.

Comments added to the files. Changes required for PR acceptance.

File: .gitignore (hunk 3 above)
Comment (luk98j) [reply .gitignore:5]: I think this is necessary

File: src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java (hunk 1 above)
Comment (luk98j) [reply src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java]: Done

File: src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java (hunk 2 above)
Comment (luk98j) [reply src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java]: Done

**Review by Przydal — APPROVED** [review]
Approved

**Pull request merged**